packages/web-client/config/environment.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
+const {
+  MERCHANT_PAYMENT_UNIVERSAL_LINK_STAGING_HOSTNAME,
+  MERCHANT_PAYMENT_UNIVERSAL_LINK_HOSTNAME,
+} = require('@cardstack/cardpay-sdk/index');
+
 const infuraIdsByTarget = {
   staging: '558ee533522a468e9d421d818e06fadb', // this infura id is specific to https://app-staging.stack.cards/
   production: '5d6efa6b750b45459184cd11dd2c8697', // this infura id is specific to https://app.cardstack.com/
 };
 
 const universalLinkHostnamesByTarget = {
-  staging: 'https://wallet-staging.stack.cards',
-  production: 'https://wallet.cardstack.com',
+  staging: MERCHANT_PAYMENT_UNIVERSAL_LINK_STAGING_HOSTNAME,
+  production: MERCHANT_PAYMENT_UNIVERSAL_LINK_HOSTNAME,
```

Comment:
The  constant is different than this previous string, does it have any impact not using `https://` ? 
The `https://` addition was done only on the `generateMerchantURL`, not in the constants, which still are:
 
`MERCHANT_PAYMENT_UNIVERSAL_LINK_STAGING_HOSTNAME = 'wallet-staging.stack.cards'`
`MERCHANT_PAYMENT_UNIVERSAL_LINK_HOSTNAME = 'wallet.cardstack.com'`